### ISSUE TITLE
run_calibfit prints error when log doesn't already exist

### ISSUE THREAD

**Issue by jacklovell** [petermilne/BOLODSP#15]
```
acq2106_061> ./bolo8_cal_cap_loop.py --cap=0 --shots=1 acq2106_061
Cycle 1
Netclient.init acq2106_061 4220
Netclient.init acq2106_061 4221
Netclient.init acq2106_061 2235
Netclient.init acq2106_061 45072
>
>
>
>
>
>
>rm: can't remove '/tmp/calibfit.log': No such file or directory
>calibration starts for channels "1"
...
```

I've had a couple of queries from users about the error message printed when trying to remove `/tmp/calibfit.log`. On first run after boot, this file doesn't exist and so doesn't need to be removed. I propose adding the `-f` flag to the `rm` command, which will silence this warning. A pull request seems like overkill for this, so if there are not objections I'll just push it straight to master.

**Comment by petermilne** [comment]
On 01/11/2019 11:37, Jack Lovell wrote:
> ```
> acq2106_061> ./bolo8_cal_cap_loop.py --cap=0 --shots=1 acq2106_061
> Cycle 1
> Netclient.init acq2106_061 4220
> Netclient.init acq2106_061 4221
> Netclient.init acq2106_061 2235
> Netclient.init acq2106_061 45072
>>
>>
>>
>>
>>
>>
>> rm: can't remove '/tmp/calibfit.log': No such file or directory
>> calibration starts for channels "1"
> ...
> ```
> 
> I've had a couple of queries from users about the error message printed when trying to remove `/tmp/calibfit.log`. On first run after boot, this file doesn't exist and so doesn't need to be removed. I propose adding the `-f` flag to the `rm` command, which will silence this warning. A pull request seems like overkill for this, so if there are not objections I'll just push it straight to master.

Go ahead. Thanks

> 


-- 
Peter Milne
Director of Sales
www.d-tacq.com

**Comment by jacklovell** [comment]
Fixed by c661083
